tests/report_tests/plugin_tests/openstack/openstack.py

```diff
@@ -0,0 +1,119 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+from sos_tests import StageOneReportTest, StageTwoReportTest, ubuntu_only
+
+
+class OpenstackPluginTest(StageOneReportTest):
+    """Basic sanity check to make sure common config files are collected
+
+    :avocado: tags=stageone
+    """
+
+    sos_cmd = ('-o openstack_cinder,openstack_keystone,openstack_glance,'
+               'openstack_neutron,openstack_nova')
```

Comment:
Would it be better to mark this as a StageTwo test and have it temporarily install a set of openstack packages? When I was first looking at OSP testing at RH, I was looking at doing a base install like we do for Foreman (i.e. the openstack tests would become their own entity under product_tests and report a new CI job) but there were some hurdles to that on single node that I'm not fully recallling at the moment.

In lieu of that, can we temporarily install some openstack packages to get some more confidence in the test? As it is, won't all of these tests pass simply due to them being missing on the test VMs?

Reply:
I hear you, and was thinking the same, at one point. 

One of the main purposes to add these as is, is to automate some the testing we at Canonical were doing at deb release time, so that we can assert them and ensure we are collecting them all automatically and spend less time collecting the data.

Happy to amend them to StageTwo, if it helps to do it here for RH OSP and Canonical OS

Reply:
I this kind of points out the discussion point from #3611

If we are doing this in stagetwo, the file will exist, hence we should have a hard assert function. This way we know 100% that we are collecting it, wdyt?

Reply:
Yeah, that's why I said I'd be onboard with a hard assert for `assertFileCollected()` and a renaming of the current method so that we have both "we absolutely 100% have this file" and "we are validating the behavior of `add_copy_spec()`'.

I'd say let's add the temporary package install for these openstack packages (I'm assuming here that the initial files get laid down by the packages, and not by some user-executed configuration script) and mark this as StageTwo.

Reply:
Cool, no worries, I'll get this re factored and tested